README.md

```
@@ -2,313 +2,1521 @@
+- [Vamsi Indugu](https://vamsiindugu.vercel.app)
+- [Vamsi Krishna Chandaluri](https://vamsi-krishna-portfolio.vercel.app)
+- [Vansh Mehta](https://vansh-mehta-portfolio.vercel.app)
+- [Varinder Singh](https://varinder148.github.io/portfolio)
+- [Varun Dey](https://varundey.me)
+- [Vatsal Patel](https://vatsalpatel.vercel.app) [Software Devekioer | Ai/Ml Engineer]
+- [Vatsal Shah](https://vatsalshah.in)
+- [Vauth](https://vauth.github.io/hueco)
+- [Vedant Athavale](https://vedantathavale.webflow.io)
+- [Vedant Milind Athavale](http://vedant-athavale.byethost31.com/portfolio.html?i=1)
+- [Vedas Dixit](https://vedas-desktop.vercel.app)
+- [Veerendra Dwivedi](https://veerendra-portfolio-seven.vercel.app)[ DevOps Engineer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://veerendra-portfolio-seven.vercel.app)